android/modules/ui/src/java/ti/modules/titanium/ui/UIModule.java

```diff
@@ -178,6 +178,9 @@
 	@Kroll.constant public static final int INPUT_TYPE_CLASS_NUMBER = InputType.TYPE_CLASS_NUMBER;
 	@Kroll.constant public static final int INPUT_TYPE_CLASS_TEXT = InputType.TYPE_CLASS_TEXT;
 
+	@Kroll.constant public static final int VIEW_GONE = View.GONE;
+	@Kroll.constant public static final int VIEW_INVISIBLE = View.INVISIBLE;
```

Comment:
Sure, changed the property and updated the constants inside the doc! `HIDDEN_BEHAVIOR_INVISIBLE` is fine, too. So its more connected to the property name